### PR TITLE
Update legendary pilgrim classes

### PR DESCRIPTION
**Pull request by ikrarkjr**
## About The Pull Request

All legendary pilgrim roles have been updated. Whether due to deprecation, or just a balance reshuffle. About half have been outright buffed. 

Roll chance for all legendary pilgrim roles have been adjusted based on combat prowess. Most likely are the weakest, least likely are the strongest.

Legendary acolyte was in the code as a file but it was blank. Now it's not! It has been added as a new legendary class.

## Why It's Good For The Game

Legendary fisher was literally straight up worse than normal fisher lmfao. Someone has to do this stuff.
You now actually FEEL legendary when you're a legendary class. No more "I'm slightly better, whoopie. Kinda nice."

### CODE REVIEW

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/Lfish.dm [hunk 1]
```diff
@@ -27,42 +27,102 @@
 		beltr = /obj/item/cooking/pan
 		mouth = /obj/item/rogueweapon/huntingknife
 		beltl = /obj/item/flint
-		backpack_contents = list(/obj/item/natural/worms = 2,/obj/item/rogueweapon/shovel/small=1)
+		backpack_contents = list(/obj/item/natural/worms/grub = 16,/obj/item/rogueweapon/shovel/small=1)
```
Comment (john-pawson) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/Lfish.dm:30]: Hey btw you can't fit 16 grubs on a satchel that only has 3 slots for things. Either change it to a normal backpack or put 2 in there.

Comment (ikrarkjr) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/Lfish.dm:30]: Lmao I thought that was a backpack, my bad let me change it to one.

**Comment by ikrarkjr** [comment]
> BTW, Female Legendary fisher still has the meme where they don't have the same equipment as male. They're missing the frying pan and flint. Also they have worse belts than a towner fisher.

Ah shit, good idea.

**Comment by challade** [comment]
Some of these weapon skills need to be tuned down. Gordon Ramsey isn't gonna be at a professional boxing coach level master in unarmed combat. Neither would a lumberjack be an expert axeman. They're both laboring folk, the skills for maiming and killing people with tools is vastly different than the skills for working with them.
This is just insane power creep.

**Comment by ikrarkjr** [comment]
> Some of these weapon skills need to be tuned down. Gordon Ramsey isn't gonna be at a professional boxing coach level master in unarmed combat. Neither would a lumberjack be an expert axeman. They're both laboring folk, the skills for maiming and killing people with tools is vastly different than the skills for working with them. This is just insane power creep.

Being legendary mortals, their abilities surpass normal humen, and their talent unmeasurable. They aren't just "laboring folk", they are legends, cut from the finest cloth.

**Comment by DorfGuy1** [comment]
BTW: playing these was almost always a massive noob trap because it had skills that were not up to date, and gear that was also out of date or not equal to their regular equivalent.

**Comment by JohnnyJohnJohnson** [comment]
> Some of these weapon skills need to be tuned down. Gordon Ramsey isn't gonna be at a professional boxing coach level master in unarmed combat. Neither would a lumberjack be an expert axeman. They're both laboring folk, the skills for maiming and killing people with tools is vastly different than the skills for working with them. This is just insane power creep.

If someone is "Legendary" for being a lumberjack, well, imagine Paul Bunyan had an estranged younger cousin, that's what someone who is LEGENDARY for being a Woodsman would be like, they should be statted and equipped appropriately.

**Pull request merged**